save-orchestrator/src/main/kotlin/org/cqfn/save/orchestrator/service/DockerService.kt

```diff
@@ -187,6 +187,7 @@ class DockerService(private val configProperties: ConfigProperties) {
                     |RUN ln -fs /usr/share/zoneinfo/UTC /etc/localtime
                     |RUN chmod +x $executionDir/$SAVE_AGENT_EXECUTABLE_NAME
                     |RUN chmod +x $executionDir/$SAVE_CLI_EXECUTABLE_NAME
+                    |RUN chmod +x $executionDir/diktat-rules/src/test/resources/test/smoke/src/ktlint
```

Comment:
may be we will hardcode it for all binaries?

Reply:
We are running the Save on this branch for testing

Reply:
Yes, we are testing a workaround. Will be implemented properly under #277